app/ante.go

```diff
@@ -2,54 +2,96 @@ package app
 
 import (
 	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
-	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	wasmTypes "github.com/CosmWasm/wasmd/x/wasm/types"
```

Comment:
why mixed case here if we use authtypes below?

I would keep consistency. wenn schon, denn schon

Reply:
👍  I had pushed a change already 🦅 👁️